src/Illuminate/Database/Eloquent/Factories/Factory.php

```diff
@@ -177,7 +177,13 @@ public function configure()
      */
     public function raw($attributes = [], ?Model $parent = null)
     {
-        return $this->state($attributes)->getExpandedAttributes($parent);
+        if ($this->count === null) {
+            return $this->state($attributes)->getExpandedAttributes($parent);
+        }
+
+        return array_map(function () use ($attributes, $parent) {
+            return $this->state($attributes)->getExpandedAttributes($parent);
+        }, range(1, $this->count));
```

Comment:
You can use Null Coalescing Operator instead of checking for `null`
```php
range(1, $this->count ?? 1)
```

Reply:
No, because that would return an array containing an array of raw attributes. If the count is `null` you just want the raw attributes themselves. This code uses the same pattern from the `make` method.